aikau/src/main/resources/alfresco/services/InfiniteScrollService.js

```diff
@@ -112,10 +130,19 @@ define(["dojo/_base/declare",
        * @returns {boolean}
        */
       nearBottom: function alfresco_services_InfiniteScrollService__nearBottom() {
-         var currentScrollPos = domGeom.docScroll().y,
-             docHeight = this.getDocumentHeight(),
-             viewport = domGeom.getContentBox(document.body).h;
-         return 0 >= (docHeight - viewport - currentScrollPos - this.scrollTolerance);
+         if (this.scrollElement)
+         {
+            var scrollPos = $(this.scrollElement).scrollTop();
```

Comment:
Originally was using a chained scrollParent() call from JQuery UI, but have stopped using that.